book/src/editor.md

```diff
@@ -32,7 +32,9 @@
 | `gutters` | Gutters to display: Available are `diagnostics` and `diff` and `line-numbers` and `spacer`, note that `diagnostics` also includes other features like breakpoints, 1-width padding will be inserted if gutters is non-empty | `["diagnostics", "spacer", "line-numbers", "spacer", "diff"]` |
 | `auto-completion` | Enable automatic pop up of auto-completion | `true` |
 | `auto-format` | Enable automatic formatting on save | `true` |
-| `auto-save` | Enable automatic saving on the focus moving away from Helix. Requires [focus event support](https://github.com/helix-editor/helix/wiki/Terminal-Support) from your terminal | `false` |
+| `auto-save.focus-lost` | Enable automatic saving on the focus moving away from Helix. Requires [focus event support](https://github.com/helix-editor/helix/wiki/Terminal-Support) from your terminal | `false` |
+| `auto-save.after-delay.enable` | Enable automatic saving after `auto-save.after-delay.timeout` milliseconds have passed since last keypress. | `false` |
+| `auto-save.after-delay.timeout` | Time in milliseconds since last keypress before auto save timer triggers. | `3000` |
```

Comment:
```suggestion
| `auto-save.after-delay.timeout` | Time in milliseconds since last edit before auto save timer triggers. | `3000` |
```

Small nit since the timer doesn't get triggered by moving

Reply:
I have also fixed the wording in some doc comments that I had forgotten to update.

Reply:
(Now DEFAULT_AUTO_SAVE_DELAY is pub again)